wagtail/admin/views/reports/locked_pages.py

```diff
@@ -34,14 +33,17 @@ class Meta:
 
 
 class LockedPagesView(PageReportView):
-    template_name = "wagtailadmin/reports/locked_pages.html"
-    title = _("Locked pages")
+    results_template_name = "wagtailadmin/reports/locked_pages_results.html"
+    page_title = _("Locked pages")
     header_icon = "lock"
     list_export = PageReportView.list_export + [
         "locked_at",
         "locked_by",
     ]
     filterset_class = LockedPagesReportFilterSet
+    index_url_name = "wagtailadmin_reports:locked_pages"
+    index_results_url_name = "wagtailadmin_reports:locked_pages_results"
+    permission_required = "unlock"
```

Comment:
Note that `permission_policy` isn't set here but it's already set in `PageReportView`